plugins/cameraserver/src/main/java/edu/wpi/first/shuffleboard/plugin/cameraserver/source/CameraUrlGenerator.java

```diff
@@ -45,31 +47,53 @@ public CameraUrlGenerator(CameraServerSource source) {
     if (frameRate > 0) {
       commands.put("fps", Integer.toString(frameRate));
     }
-    return generateUrls(commands, baseUrls);
+    commands.put("name", source.getName());
+    return generateUrls(commands, baseUrls, source.getName());
   }
 
   @VisibleForTesting
-  static String[] generateUrls(Map<String, String> commands, String[] baseUrls) { // NOPMD varargs instead of array
+  static String[] generateUrls(Map<String, String> commands, String[] baseUrls,
+                               String cameraName) { // NOPMD varargs instead of array
```

Comment:
Don't need the NOPMD comment if `baseUrls` is no longer the final method parameter

Reply:
Fixed